cmp/grid/columns/Column.js

```diff
@@ -548,8 +562,20 @@ export class Column {
     //--------------------
     // Implementation
     //--------------------
-    defaultComparator = (v1, v2) => {
-        const sortCfg = find(this.gridModel.sortBy, {colId: this.colId});
+    defaultComparator = (v1, v2, agNodeA, agNodeB) => {
+        const {gridModel, colId, sortValue} = this,
+            sortCfg = find(gridModel.sortBy, {colId}),
+            recordA = agNodeA?.data,
+            recordB = agNodeB?.data;
+
+        if (isString(sortValue)) {
+            v1 = recordA?.data[sortValue] ?? v1;
+            v2 = recordB?.data[sortValue] ?? v2;
+        } else if (isFunction(sortValue)) {
+            v1 = sortValue(v1, {recordA, column: this, gridModel});
+            v2 = sortValue(v2, {recordB, column: this, gridModel});
+        }
```

Comment:
This code block is very similar to the one above. Could we DRY these up with a `getSortValue(record)` method?

Reply:
yes good idea.